Assets/Scripts/Network/ColyseusClient.cs

```diff
@@ -93,142 +91,123 @@ public async void ConnectToServer(string serverEndpoint, string username, string
                 networkController.ServerStatusMessage = ex.Message;
                 networkController.RefreshUI();
             }
-            
         }
     }   
     
-    public async Task Disconnect()
+    public void Disconnect()
     {
         if (IsConnected)
         {
-            await LeaveRoom();
+            LeaveRoom();
             if (players != null)    
             {
                 players.Clear();
             }
-            client.Auth.Logout();
+            // client.Auth.Logout();

```

Comment:
I usually recommend removing commented out sections like this, but I guess knowing where authentication actions used to happen will help us if/when we implement some other auth strategy.